tests/LoginTest.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+require_once 'login.php';
+
+use PHPUnit\Framework\TestCase;
+
+class LoginTest extends TestCase {
+
+    public function test_loginTest_userDoesntExist()
+    {
+        $expected = 'user does not exist';
+
+        $db = connectToDb();
+        $usersModel = new UsersModel($db);
+
+        $inputtedEmail = 'Toni@gmail.com';
+        $actual = '';
+        $user = $usersModel->selectUser($inputtedEmail);
+        if ($user === null) {
+            $actual = 'user does not exist';
+        }
+
+        $this->assertEquals($expected, $actual);
+    }
```

Comment:
this is an integration test, not a unit test. cannot unit test things that connect to the DB. you should delete this